sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -328,8 +328,10 @@ struct _pi_queue : _pi_object {
   // Attach a command list to this queue and allow it to remain open
   // and used for further batching.  It may be executed immediately,
   // or it may be left open for other future command to be batched into.
+  // OKToBatchKernel indicates whether for this particular kernel in the
```

Comment:
Even though it is currently only used for `piEnqueueKernelLaunch`, it is created a more general and it is not clear what the "kernel" is here then.

Reply:
I could change the name to OKToBatchThisCommand?  Would you like it better if I changed this to be a parameter to executeCommandList, defaulted OKToBatchThisCommand tp false, and moved the batching logic to executeCommandList.

Reply:
I think so, yes.

Reply:
Done